index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
```

Comment:
The second h2 statement is not being executed making it irrelevant. Change the heading for the second h2 to maybe h1 or h3 to distinguish between the two when calling. Then change the returned text to "Game over, you win. Reload to try again." If the user wins the game.
The pull request does not account for the script written as no fault or error was found.
The author created a well-built game that runs perfectly and terminates whenever the player either loses or wins. The author should have highlighted when a player loses or wins on the output message.